serverless/functions/verify-passcode.js

```diff
@@ -0,0 +1,18 @@
+/* global Twilio Runtime */
+'use strict';
+
+// verifies that auth.js does not throw error for passcode:
+
+module.exports.handler = async (context, event, callback) => {
+  const authHandler = require(Runtime.getAssets()['/auth.js'].path);
+  authHandler(context, event, callback);
+
+  let response = new Twilio.Response();
+  response.appendHeader('Content-Type', 'application/json');
+
+  response.setStatusCode(200);
+
+  response.setBody({});
```

Comment:
Not sure this will actually mean anything, but I'm wondering if we could do something like this:
`response.setBody({ verified: true });`

Some of the other endpoints here have something similar.